botocove/cove_decorator.py

```diff
@@ -173,6 +192,7 @@ def cove(
     ignore_ids: Optional[List[str]] = None,
     rolename: Optional[str] = None,
     role_session_name: Optional[str] = None,
+    session_policy: Optional[str] = None,
```

Comment:
I wonder if we could overload this to allow passing a Dict too, or if there's a nicer way to pass a policy than stringified json? Maybe excessive if the API only takes json strings

Reply:
because boto just takes a JSON string, I would expect botocove to do the same. Anything more probably isn't worth the complexity.

I imagine an application using botocove may even store its policies in .json files to to take advantage of linters like [parliament](https://github.com/duo-labs/parliament). At runtime the policy would just be read as string and passed on.

Reply:
sure - makes sense. Not functionality I've ever needed so unfamiliar but I understand the use :)